src/utils/constants.ts

```diff
@@ -32,7 +32,8 @@ export const COMPANY_NAME_PLACEHOLDER = "COMPANY_NAME_PLACEHOLDER";
 export const TRANSACTION_ID_PLACEHOLDER = "TRANSACTION_ID_PLACEHOLDER";
 export const APPOINTMENT_ID_PLACEHOLDER = "APPOINTMENT_ID_PLACEHOLDER";
 export const CONSENT_TO_ACT_AS_DIRECTOR = "Select if you confirm that by submitting this information, the person named has consented to act as director";
-
+export const SA_TO_ROA_ERROR = "Select yes if the registered office address changes in the future, and you want this to apply to your correspondence address too";
+export const HA_TO_SA_ERROR = "Select yes if the home address changes in the future, and you want this to apply to the correspondence address";
```

Comment:
I might be wrong here but shouldn't this read:
"_Select yes if the correspondence address changes in the future, and you want this to apply to the home address_"

Reply:
Will check with product